packages/core/src/indexing/postgres/postgres-index-api.ts

```diff
@@ -53,8 +54,13 @@ export class PostgresIndexApi implements DatabaseIndexApi {
     return this.insertionOrder.page(query)
   }
 
+  async verify(validTableStructure: Object): Promise<void> {
+    await verifyTables(this.dbConnection, this.modelsToIndex, validTableStructure)
```

Comment:
we probably also want to verify that the database indexes we expect are present too.  But let's just file that as a `debt` ticket and leave it for later, doesn't seem critical to do now